### ISSUE TITLE
Shell script failed but I can't see why

### ISSUE THREAD

**Issue by samskiter** [supermarin/xcpretty#239]
So I've been getting an error similar to this, but only on my build machine: https://github.com/fastlane/fastlane/issues/4197#issuecomment-215088105

It happens only when I run through jenkins. If I execute my build on the build machine, from the jenkins working directory, the build succeeds.

It's always failing at the same spot but unfortunately xcpretty seems to be hiding the error output from the pods shell script (as shown in the linked comment).

Is there any way to get the output without running xcodebuild directly (which generates a HUGE output)

**Comment by jnewc** [comment]
I also have this issue. In my case it appears to be hiding issues to do with code signing / missing provisioning profiles. The output I receive before failure: 

```
▸ Building [REMOVED]
▸ Check Dependencies
** ARCHIVE FAILED **
```

I can see the real issue when running xcodebuild so it's clear that xcpretty is omitting the cause.

**Comment by supermarin** [comment]
stay tuned, this will get released/fixed very soon

**Comment by samskiter** [comment]
@supermarin Fixed now?

**Comment by supermarin** [comment]
You can try updating to the latest 0.2.x release, altho the proper fixes will land in 0.3.x onwards
